utils/FTplots.m

```diff
@@ -118,13 +118,13 @@
 
 if (~isempty(reference) && ~byChannel)
     for i=1:size(fields,1)
-        figure,
+        figure('WindowStyle','docked'),
         plot(time-timeStampinit,data.(fields{i})(:,1),xPlotOptions);hold on;
         plot(time-timeStampinit,data.(fields{i})(:,2),yPlotOptions);hold on;
         plot(time-timeStampinit,data.(fields{i})(:,3),zPlotOptions);hold on;
-        plot(time-timeStampinit,reference.(fields{i})(:,1),x2PlotOptions);hold on;
-        plot(time-timeStampinit,reference.(fields{i})(:,2),y2PlotOptions);hold on;
-        plot(time-timeStampinit,reference.(fields{i})(:,3),z2PlotOptions);hold on;
+        plot(time-timeStampinit,reference.(fields{1})(:,1),x2PlotOptions);hold on;
```

Comment:
why fields was change to 1? fields should change accordingly for both data and reference since its assumed that both can have multiple sensors which are thenames of the fields in this case you are comparing incorrectly the sensors
since you would be comparing the sensor in the right leg to the right arm of a different dataset

Reply:
For some reason, `reference` was not being populated with as many fields as in `data`. I have to try again...